cmd/feed-ingress/main.go

```diff
@@ -43,6 +44,7 @@ var (
 	nginxConfig                    nginx.Conf
 	nginxLogHeaders                cmd.CommaSeparatedValues
 	nginxTrustedFrontends          cmd.CommaSeparatedValues
+	nginxDefaultSslPath            string
```

Comment:
This is inconsistent capitalisation - you have HTTPS for an acronym above, but Ssl here